specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/ServiceFeatureOCITask.json

```diff
@@ -0,0 +1,142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Database Migration Service Resource Provider",
+    "version": "2018-07-15-preview"
```

Comment:
Are all these definitions part of this version or have they been copied over from an older version?

Reply:
These are new features which were not available before.  We have no breaking changes so I kept the version.